util/package.go

```diff
@@ -50,6 +50,8 @@ type Package struct {
 	Datasources   []Datasource `config:"datasources,omitempty" json:"datasources,omitempty" yaml:"datasources,omitempty"`
 	Download      string       `json:"download" yaml:"download,omitempty"`
 	Path          string       `json:"path" yaml:"path,omitempty"`
+	// This is in purpose `dataset.types` with an "s" as it is a summary of the existing dataset types
+	DatasetTypes []string `json:"dataset.types" yaml:"dataset.types,omitempty"`
```

Comment:
should both ones have the `omitempty` annotation?

Reply:
yes, and I realised that we don't need the yaml part as it never shows up in yaml.